jira/client.py

```diff
@@ -196,7 +197,7 @@ class JIRA(object):
     AGILE_BASE_URL = GreenHopperResource.AGILE_BASE_URL
 
     def __init__(self, server=None, options=None, basic_auth=None, oauth=None, jwt=None, kerberos=False,
-                 validate=False, get_server_info=True, async=False, logging=True, max_retries=3, proxies=None):
+                 validate=False, get_server_info=True, async=False, max_retries=3, proxies=None):
         """Construct a JIRA client instance.
```

Comment:
I am investigating now a possible migration to `kwargs` as I would prefer not to break the existing code that is using the logging parameter. If we would use kwargs we could avoid breaking the code, old code will continue to work.

I am quite busy and it is easy to miss a notification, also I don't use this library at work which makes is less likely to get attention.